lib/mix/tasks/triplex.mysql.install.ex

```diff
@@ -27,7 +27,7 @@ defmodule Mix.Tasks.Triplex.Mysql.Install do
         Mix.raise "the tenant table only makes sense for MySQL repositories"
       end
 
-      path = Path.relative_to(Mix.Ecto.migrations_path(repo),
+      path = Path.relative_to(Mix.EctoSQL.ensure_migrations_path(repo),
```

Comment:
Well, I see there is no more a `migrations_path` function on the new `ecto_sql` lib, I think it's cool anyway.

Reply:
This is private API and shouldn't be used by external tools. Use `Ecto.Migrator.migrations_path/1` instead.